src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/services.component.ts

```diff
@@ -178,6 +185,10 @@ export class ServicesComponent extends ListWithDetails implements OnChanges, OnI
       this.orchStatus = status;
       this.showDocPanel = !status.available;
     });
+
+    this.configureServiceUrl('api/grafana/url', 'grafana');
+    this.configureServiceUrl('api/prometheus/url', 'prometheus');
+    this.configureServiceUrl('api/prometheus/alertmanager_url', 'alertmanager');
```

Comment:
```suggestion
    this.configureServiceUrl('api/grafana/url', 'grafana');
    this.configureServiceUrl('ui-api/prometheus/prometheus-api-host', 'prometheus');
    this.configureServiceUrl('ui-api/prometheus/alertmanager-api-host', 'alertmanager');
```

Reply:
Completly missed it, thanks for pointing it out